nilearn/masking.py

```diff
@@ -254,8 +254,6 @@ def compute_epi_mask(epi_img, lower_cutoff=0.2, upper_cutoff=0.85,
     if verbose > 0:
         print("EPI mask computation")
 
-    epi_img = _utils.check_niimg(epi_img)
```

Comment:
> > -    epi_img = _utils.check_niimg(epi_img)

> This is clever but your efforts are kinda wasted by the new_img_like at line 289/287.

Not too bad, because if it's a 4D image, new_img_like doesn't call
'get_data', and so the data is not loaded.

That said, new_img_like is a bit of a tangle, as it uses load_img, which
itself uses new_img_like. new_img_like should probably not be using
load_img, and maybe nibabel directly.

Would you be OK to special-case new_img_like to use nibabel directly,
and, as a consequence, not load the data for list of string?